### ISSUE TITLE
Use compass mixins when possible

### ISSUE THREAD

**Issue by ericdfields** [vwall/compass-twitter-bootstrap#28]
Since the name of the project assumes a dependency on compass, why not cut down on some code by using pre-existing mixins for things like box-shadow instead of emulating twitter's less functions?

I did the same in another port, not realizing this one was already so mature ;-) https://github.com/hecbuma/compass-bootstrap

**Comment by vwall** [comment]
That's how this project started out, I was just in a hurry to push out 2.0.  I would like to make the parser do this.  It's really no fun doing it by hand.

**Comment by aaronchi** [comment]
Yep. default compass box-shadow mixin is getting overwritten and causing problems.

**Comment by wyuenho** [comment]
background-size is getting overwritten too. It maybe wise to prefix all compass-twitter-bootstrap mixins mixin includes to avoid future name conflicts again.

**Comment by tijsverkoyen** [comment]
see https://github.com/vwall/compass-twitter-bootstrap/pull/87

**Comment by ldexterldesign** [comment]
Ditto, comment out the Compass Bootstrap box-shadow() @mixin and revert back to Compass' version, which works.

Hope to see this fixed in the next release.

Kind regards,

PS Love you work here on the port - keep it up!
